.github/workflows/update-prometheus.yml

```diff
@@ -0,0 +1,72 @@
+name: Update Prometheus
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - "main"
+    paths:
+      - "prometheus/**"
+
+jobs:
+  build-prometheus-image:
+    name: Build Docker image and push to ECR
+    environment: dev
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Install GitVersion
+        uses: gittools/actions/gitversion/setup@v0.10.2
+        with:
+          versionSpec: '5.x'
+
+      - name: Determine Version
```

Comment:
Are we versioning the prometheus image?

Reply:
Will remove that here and for grafana, missed that when going through it